tests/unit/includes/gateway/class-omise-payment-ocbc-digital-test.php

```diff
@@ -0,0 +1,111 @@
+<?php
+
+require_once __DIR__ . '/class-omise-offsite-test.php';
+
+class Omise_Payment_OCBC_Digital_Test extends Offsite_Test
+{
+    private $obj;
+
+    public function setUp(): void
+    {
+        parent::setUp();
+        require_once __DIR__ . '/../../../../includes/gateway/class-omise-payment-ocbc-digital.php';
+        require_once __DIR__ . '/../../../../includes/classes/class-omise-image.php';
+        $this->obj = new Omise_Payment_OCBC_Digital();
+    }
+
+    public function tearDown(): void
+    {
+        // destroy object and clear memory
+        unset($this->obj);
+    }
+
+    /**
+     * @test
```

Comment:
Don't we need to add `@covers` for coverage report?

Reply:
I thought so too but I guess it is not needed because Sonarcloud is picking up coverage without it.